proxy_ldap.conf.template

```diff
@@ -1,3 +1,7 @@
+$([[ -v NO_VERIFY_SERVER_CERT ]] && { echo "
```

Comment:
I think this block should be placed after the `LoadModule ldap_module` line , possibly just before the
```
$([[ -v LDAPS_CACERT_PEM ]] && { echo "
```